sample-geovelocity/siddhi-execution-geovelocity/src/main/java/org/wso2/extension/siddhi/execution/geovelocity/CheckRestrictedAreas.java

```diff
@@ -0,0 +1,178 @@
+                        "combinations considering the previous login and the current login attempt." +
+                        "Risk value which is based on restricted location combination is either 1 or 0." +
+                        "If the current and previous login location is according to the predefined " +
+                        "restricted location combinations, restricted area based risk value is 1, " +
+                        "otherwise it's 0.",
+                type = {DataType.INT}),
+        examples = @Example(
+                description = "This will return risk score based on the restricted area " +
+                        "combinations considering the locations of the previous login and the current login" +
+                        " attempt. Assume Alex previous login location is South Korea and current login " +
+                        "location is North Korea. Since it's defined that it's restricted to travel from " +
+                        "South Korea to North Korea, risk is 1",
+                syntax = "define stream GeovelocityStream(current.login.city string, " +
+                        "previous.login.city string, current.login.country string, previous.login.country string);" +
+                        "from GeovelocityStream" +
+                        "select geovelocity:restrictedAreaBasedRisk(current.login.city, previous.login.city, " +
+                        "current.login.country, previous.login.country) as restrictedareabasedrisk" +
+                        "insert into outputStream;")
+)
+
+public class CheckRestrictedAreas extends FunctionExecutor {
+
+    private static final String DEFAULT_GEOVELOCITY_RESOLVER_CLASSNAME =
+            "org.wso2.extension.siddhi.execution.geovelocity.internal.impl.DefaultDBBasedGeoVelocityDataResolver";
+    private static GeoVelocityDataResolver geoVelocityDataResolverImpl;
+
+    @Override
+    protected void init(ExpressionExecutor[] attributeExpressionExecutors, ConfigReader configReader,
+                        SiddhiAppContext siddhiAppContext) {
+
+        initializeExtensionConfigs(configReader);
+        if (attributeExpressionExecutors.length != 4) {
+            throw new SiddhiAppValidationException("Invalid no of arguments passed to " +
+                    "geovelocity:restrictedareabasedrisk function, required 4, but found "
+                    + attributeExpressionExecutors.length);
+        }
+        Attribute.Type attributeType = attributeExpressionExecutors[0].getReturnType();
+        if (attributeType != Attribute.Type.STRING) {
+            throw new SiddhiAppValidationException("Invalid parameter type found for first " +
+                    "argument current.login.city of geovelocity:loginbehaviourbasedrisk() function, required "
+                    + Attribute.Type.STRING + ", but found " + attributeType.toString());
+        }
+        Attribute.Type secondattributeType = attributeExpressionExecutors[1].getReturnType();
```

Comment:
Shall we rename these variables in camel case, `secondattributeType`, `thirdattributeType`, `fourthattributeType`